.github/workflows/agent-executor.yml

```
@@ -0,0 +1,342 @@
+    name: Complete Agent Work
+    needs: [setup, plan, execute]
+    if: always()
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Update Issue Status
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "📝 Updating issue #${{ inputs.issue_number }} status..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)